packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+
+#### Metrics Configuration
+
+| Option Name            | Type    | Description                                                                                                                                                                                                                                                                                                                                              | Default                       | Required?                |
+| ---------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- | ------------------------ |
+| `metricsSubmissionUrl` | String  | Metrics server hostname. By default the value is set to https://events.backtrace.io.                                                                                                                                                                                                                                                                     | `https://events.backtrace.io` | <ul><li>- [ ] </li></ul> |
+| `enable`               | Boolean | Determines if the metrics support is enabled. By default the value is set to true.                                                                                                                                                                                                                                                                       | `true`                        | <ul><li>- [ ] </li></ul> |
+| `autoSendInterval`     | Number  | Indicates how often crash free metrics are sent to Backtrace. The interval is a value in ms. By default, session events are sent on application startup/finish, and every 30 minutes while the application is running. If the value is set to 0. The auto send mode is disabled. In this situation the application needs to maintain send mode manually. | On application startup/finish | <ul><li>- [ ] </li></ul> |
+| `size`                 | Number  | Indicates how many events the metrics storage can store before auto submission.                                                                                                                                                                                                                                                                          | `50`                          | <ul><li>- [ ] </li></ul> |
+
+#### Metrics Usage
+
+```ts
+// metrics will be undefined if not enabled
+client.metrics?.send();
+```
+
+---
+
+### Offline database support
+
+The Backtrace Electron SDK can cache generated reports and crashes to local disk before sending them to Backtrace. This
```

Comment:
I think it's worth to mention we're using crash reporting solution available in electron. If the user overrides it, crashes might not be available in Backtrace

Reply:
This is specified in `Native crash support`.